src/Ng/Models/RegistrationIndexPackage.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Ng.Models
+{
+    /// <summary>
+    /// Model package registration information contained in the registartion index, 
+    /// https://api.nuget.org/v3/registration1/newtonsoft.json/index.json#items#items#items.
+    /// </summary>
+    /// <remarks>The registration index contains all the versions of a package. This type 
+    /// contains the registration information for one version of the package.</remarks>
+    public class RegistrationIndexPackage
```

Comment:
Yes, it probably should inherit RegistrationItem. They're the same. However, I meant to remove RegistrationItem from the project. I don't think it's used anymore. I'll just delete RegistrationItem.